frontend/packages/kubevirt-plugin/src/components/vms/vm.scss

```diff
@@ -1,4 +1,8 @@
-.kubevirt-vm-list {
```

Comment:
the naming is `.ns-component__elem--modifier` so here it should be `.kv-vm-list`

ref: http://getbem.com/naming/ + console code